src/EoS/TaubMathews/CPU_EoS_TaubMathews.cpp

```diff
@@ -0,0 +1,336 @@
+#include "CUFLU.h"
+#ifdef __CUDACC__
+#include "CUAPI.h"
+#include "CUFLU_Shared_FluUtility.cu"
+#endif
+
+#if ( MODEL == HYDRO && defined SRHD )
+
+
+/********************************************************
+1. Approximately relativistic ideal gas EoS (Taub-Mathews EoS)
+
+2. This file is shared by both CPU and GPU
+
+   GPU_EoS_TaubMathews.cu -> CPU_EoS_TaubMathews.cpp
+
+3. Three steps are required to implement an EoS
+
+   I.   Set an EoS auxiliary array
+   II.  Implement EoS conversion functions
+   III. Set EoS initialization functions
+********************************************************/
+
+
+
+// =============================================
+// I. Set an EoS auxiliary array
+// =============================================
+
+//-------------------------------------------------------------------------------------------------------
+// Function    :  EoS_SetAuxArray_TaubMathews
+// Description :  Set the auxiliary array AuxArray[]
+// Note        :  1. Invoked by EoS_Init_TaubMathews()
+//                2. AuxArray[] has the size of EOS_NAUX_MAX defined in Macro.h (default = 10)
+//                3. Add "#ifndef __CUDACC__" since this routine is only useful on CPU
+//                4. Do not change the order of AuxArray[]
+//                   --> For example, the dual-energy routines assume AuxArray[0]=GAMMA
+//
+// Parameter   :  AuxArray : Array to be filled up
+//
+// Return      :  AuxArray[]
+//-------------------------------------------------------------------------------------------------------
+#ifndef __CUDACC__
+void EoS_SetAuxArray_TaubMathews( double AuxArray_Flt[], int AuxArray_Int[] )
+{
+   AuxArray_Flt[0] = ( OPT__UNIT ) ? MOLECULAR_WEIGHT * Const_amu / Const_kB * (UNIT_E/UNIT_M)
```

Comment:
Replace `Const_amu` with `MU_NORM`, which has a default value of `Const_mH` instead of `Const_amu` to be more consistent with the gamer frontend in yt (though the values of `Const_mH` in gamer and yt are still not fully consistent).